src/coprocessor/dag/rpn_expr/impl_arithmetic.rs

```diff
@@ -313,6 +313,55 @@ impl ArithmeticOp for DecimalMultiply {
     }
 }
 
+#[rpn_fn]
+#[inline]
+fn abs_int(arg: &Option<Int>) -> Result<Option<Int>> {
```

Comment:
Do we need to move these functions into `impl_math.rs`?
/cc @breeswish

Reply:
Nice catch. Please move it to `impl_math` since it is in `builtin_math` in TiDB.

Reply:
> Do we need to move these functions into `impl_math.rs`?
> /cc @breeswish

Done